exercises/more_string_manipulation/challenge_080.py

```diff
@@ -0,0 +1,40 @@
+# MORE STRING MANIPULATION
+
+# Challenge 080
+"""
+Ask the user to enter their first name and then display the length of their first name.
+Then ask for their surname and display the length of their username. Join their first name
+and surname together with a space between and display the result.
+Finally, display the length of their full name (including the space).
+"""
+
+
+def processing(first_name, last_name):
+    full_name = first_name + " " + last_name
```

Comment:
Why did you not use a formatted string here?

Reply:
Changed it!